doc/classes/Dictionary.xml

```diff
@@ -291,6 +291,14 @@
 				Returns the list of keys in the [Dictionary].
 			</description>
 		</method>
+		<method name="merge">
+			<return type="void" />
+			<argument index="0" name="dictionary" type="Dictionary" />
+			<argument index="1" name="overwrite" type="bool" default="false" />
+			<description>
+				Adds elements from [code]dictionary[/code] to this [Dictionary]. If [code]overwrite[/code] is [code]false[/code], keys that already exist will not be added.
```

Comment:
Without looking at the prototype, the "if" part sounds like `overwrite` is `true` by default.

Reply:
I wasn't sure how to word it for `true`. I mean something like:
```
If overwrite is true, duplicate keys will have their value overwritten.
```
doesn't really say that with `false` these keys will be skipped. And describing both cases is a bit redundant, idk.

Reply:
What about something like:

> Adds elements from `dictionary` to this `Dictionary`. Elements with duplicated keys are overwritten by default. To keep existing elements, set `overwrite` to `false`.